src/pytkdocs/loader.py

```diff
@@ -415,6 +431,49 @@ def get_property_documentation(self, node: ObjectNode) -> Attribute:
             source=source,
         )
 
+    def get_pydantic_field_documentation(self, node: ObjectNode) -> Attribute:
+        """
+        Get the documentation for a PyDantic Field
+
+        Arguments:
+            node: The node representing the Field and its parents.
+
+        Return:
+            The documented attribute object.
+        """
+        prop: ModelField = node.obj
+        path = node.dotted_path
+        properties = ["field"]
```

Comment:
Maybe `pydantic-field` to be even more explicit? I'm thinking about users reading documentation generated by mkdocstrings, `field` would be a bit ambiguous, don't you think?

We could also think about adding a `pydantic-model` property to the class itself (not in this function of course)? Do you think it's a good idea?

Reply:
I was thinking of just adding the `pydantic` tag to properties for both.

Reply:
OK!